osidb/serializer.py

```diff
@@ -583,6 +583,38 @@ class TrackerSerializer(
 ):
     """Tracker serializer"""
 
+    jira_issuetype = serializers.CharField(
```

Comment:
You can use [`ChoiceField`](https://www.django-rest-framework.org/api-guide/fields/#choicefield) so you don't have to check the values below in the `is_valid` method

Reply:
Valid point and no longer an issue. Removed the ability to set issuetype via API in a new commit.